---

input_file: "pcd_images/shape1_shape2/shape2_0deg.pcd"
output_file: "pcd_images/shape1_shape2/shape2_0deg_filtered.pcd"
save_output: true

filter_box: [-0.5, 0.5, -1.0, -0.25, 0.005, 0.25] # bounding box limits

voxel_leaf_size: 0.005                           # voxel leaf size
